roles/kubernetes/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+- name: install kubernetes tools
+  homebrew:
+    name: kubectl
+    state: present
+
+- name: install kubeclj
+  git:
+    repo: ssh://git@github.com/Guaranteed-Rate/kubeclj.git
```

Comment:
I was planning to leave GR stuff out of this and leave it more of a generic development machine. The kubectl above is fine, though.